client/context/context.go

```diff
@@ -84,7 +87,7 @@ func (ctx CLIContext) WithAccountStore(accountStore string) CLIContext {
 // WithFromAddressName returns a copy of the context with an updated from
 // address.
 func (ctx CLIContext) WithFromAddressName(addrName string) CLIContext {
```

Comment:
I think we'd want to search for "FromAddressName" and replace them w/ "From" as well, like this method name.